priv/tools/internal/riak_cs_inspector.erl

```diff
@@ -259,41 +261,110 @@ cs_bucket_info(RiakBucket, KnownNames) ->
             ['riak-cs', RiakBucket, RiakBucket]
     end.
 
--spec list_cs_buckets(pid()) -> any().
-list_cs_buckets(RiakcPid)->
+list_cs_buckets(RiakcPid, ListFun)->
     heading("~-64..=s ~-8..=s: ~-40..=s~n",
             ["CS Bucket Name ", "Sibl. ", "Owner Key "]),
-    {ok, Keys} = riakc_pb_socket:list_keys(RiakcPid, "moss.buckets"),
-    lists:foreach(fun(Key) ->
-                          [io:format("~-64s ~-8B: ~-40s~n", [Key, SiblingNo, V])
-                           || {SiblingNo, {_MD, V}}
-                                  <- get_riak_object(RiakcPid, "moss.buckets", Key)]
-                  end, lists:sort(Keys)).
+    PrintFun = fun(Keys) ->
+                       lists:foreach(fun(Key) ->
+                                             [io:format("~-64s ~-8B: ~-40s~n", [Key, SiblingNo, V])
+                                              || {SiblingNo, {_MD, V}}
+                                                     <- get_riak_object(RiakcPid, "moss.buckets", Key)]
+                                     end, Keys)
+               end,
+    ListFun(RiakcPid, "moss.buckets", PrintFun).
+
+lookup_listbucket_fun(Opts) ->
```

Comment:
It's better to place this function and `lookup_listkey_fun/1` after `%% Riak Utilities` comment line.